diff --git a/dist/GraphiQL.d.ts b/dist/GraphiQL.d.ts
index 72a65d6b3f4fd60ecee13973909b16a33649d419..8a729c82fa1e2353514d075549f302e75a9df23f 100644
--- a/dist/GraphiQL.d.ts
+++ b/dist/GraphiQL.d.ts
@@ -16,7 +16,7 @@ declare type QueryEditorProps = ComponentPropsWithoutRef<typeof QueryEditor>;
 declare type VariableEditorProps = ComponentPropsWithoutRef<typeof VariableEditor>;
 declare type HeaderEditorProps = ComponentPropsWithoutRef<typeof HeaderEditor>;
 declare type ResponseEditorProps = ComponentPropsWithoutRef<typeof ResponseEditor>;
-interface GraphiQLInterfaceProps extends EditorProps, AddSuffix<Pick<QueryEditorProps, 'onEdit'>, 'Query'>, AddSuffix<Pick<VariableEditorProps, 'onEdit'>, 'Variables'>, AddSuffix<Pick<HeaderEditorProps, 'onEdit'>, 'Headers'>, Pick<ResponseEditorProps, 'responseTooltip'>, Pick<ComponentPropsWithoutRef<typeof Sidebar>, 'forcedTheme' | 'showPersistHeadersSettings'> {
+export interface GraphiQLInterfaceProps extends EditorProps, AddSuffix<Pick<QueryEditorProps, 'onEdit'>, 'Query'>, AddSuffix<Pick<VariableEditorProps, 'onEdit'>, 'Variables'>, AddSuffix<Pick<HeaderEditorProps, 'onEdit'>, 'Headers'>, Pick<ResponseEditorProps, 'responseTooltip'>, Pick<ComponentPropsWithoutRef<typeof Sidebar>, 'forcedTheme' | 'showPersistHeadersSettings'> {
     children?: ReactNode;
     /**
      * Set the default state for the editor tools.
@@ -61,4 +61,5 @@ export declare const GraphiQL: FC<GraphiQLProps> & {
         children: ReactNode;
     }>;
 };
+export declare const GraphiQLInterface: FC<GraphiQLInterfaceProps>;
 export {};
diff --git a/dist/GraphiQL.js b/dist/GraphiQL.js
index 3c975882b6cde6bda8b6e08d9950ffd2afc3bf23..7135bfcd16df55533931e496cb15a9a0aebc2020 100644
--- a/dist/GraphiQL.js
+++ b/dist/GraphiQL.js
@@ -855,6 +855,7 @@ function _temp(acc, curr) {
   return acc;
 }
 export {
-  GraphiQL
+  GraphiQL,
+  GraphiQLInterface
 };
 //# sourceMappingURL=GraphiQL.js.map
diff --git a/dist/index.d.ts b/dist/index.d.ts
index 0d3c523b26d1bc1505197a89d5f3c01c938e930a..9b385897612ccc9a2a0803d4dd8ac6bcae7302b9 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -1,2 +1,2 @@
-export { GraphiQL, type GraphiQLProps } from './GraphiQL';
+export { GraphiQL, GraphiQLInterface, type GraphiQLProps, type GraphiQLInterfaceProps } from './GraphiQL';
 export { HISTORY_PLUGIN } from '@graphiql/plugin-history';
diff --git a/dist/index.js b/dist/index.js
index 2f0878ecfdad654ebd39339436ab3b2be6e60355..7bc47162db81be0458afc9cfff14864d661903da 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1,8 +1,9 @@
 /* empty css          */
-import { GraphiQL } from "./GraphiQL.js";
+import { GraphiQL, GraphiQLInterface } from "./GraphiQL.js";
 import { HISTORY_PLUGIN } from "@graphiql/plugin-history";
 export {
   GraphiQL,
+  GraphiQLInterface,
   HISTORY_PLUGIN
 };
 //# sourceMappingURL=index.js.map
